pkg/agent/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -280,3 +321,76 @@ func TestDeleteRule(t *testing.T) {
 		t.Fatal("Expected one update, got none")
 	}
 }
+
+func TestNetworkPolicyWithMutipleRules(t *testing.T) {
```

Comment:
```suggestion
func TestAddNetworkPolicyWithMutipleRules(t *testing.T) {
```
to keep same test name style